browsergym/experiments/src/browsergym/experiments/loop.py

```diff
@@ -173,7 +174,6 @@ def run(self):
             )
             logger.debug(f"Environment created.")
 
-            err_msg, stack_trace = None, None
```

Comment:
This was not needed anymnore?

Reply:
It's still needed, I moved it up before the try loop so that the variables are always defined when the `finally` executes.